pkg/cluster/k8sres_test.go

```diff
@@ -788,16 +795,31 @@ func TestPodEnvironmentSecretVariables(t *testing.T) {
 			subTest: "Secret referenced by PodEnvironmentSecret does not exist",
 			opConfig: config.Config{
 				Resources: config.Resources{
-					PodEnvironmentSecret: "idonotexist",
+					PodEnvironmentSecret:  "idonotexist",
+					ResourceCheckInterval: time.Duration(3),
+					ResourceCheckTimeout:  time.Duration(10),
+				},
+			},
+			err: fmt.Errorf(`could not read Secret PodEnvironmentSecretName: still failing after 3 retries: secret.core "idonotexist" not found`),
```

Comment:
```suggestion
			err: fmt.Errorf("could not read Secret PodEnvironmentSecretName: still failing after 3 retries: secret.core %q not found", "idonotexist"),
```

Reply:
Would make sense to have a constant for the name, check #1794

Reply:
I've added [this commit](https://github.com/zalando/postgres-operator/pull/1777/commits/4ea0954efcfda03010a800148436375cdd17524c), is that what you were looking for?